modules/node_modules/@ncigdc/components/CurrentFilters.js

```diff
@@ -130,7 +132,7 @@ const CurrentFilters = ({
         <Row wrap spacing="0.3rem">
           <NotUnderlinedLink
             style={styles.groupPadding}
-            query={{}}
+            query={{ searchTableTab }}
```

Comment:
It should probably keep other things like the facet tab as well.
Might be better to specify the things we are clearing instead? either by passing in the whole query and omitting filters or making `query={{filters: null}}` work.